src/codeswitch/data_generation/synthatic_data_generation.py

```diff
@@ -0,0 +1,160 @@
+import os
+import random
+import re
+import time
+from collections import Counter
+
+import pandas as pd
+from openai import OpenAI
+from tqdm import tqdm
+
+# pylint: disable=E1101
+
+# Initialize the OpenAI client
+api_key = os.getenv("open_ai_key")
```

Comment:
Dont overwrite the default behavior of openai api_key. This causes incompatibility with configuration of other users.